src/style.css

```diff
@@ -1 +1,136 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(fonts/Roboto-Medium.woff);
+}
+
+.html {
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+}
+
+/*
+  Чому я використовую бокс-сайзінг таким чином,
+  замість звичайного * {...}, тому що деякі бібліотеки,
+  по типу Swiper, використовують box-sizing: content-box
+  (наприклад класс swiper-wrapper) і, щоб бібліотека відображалась так,
+  як це задумував розробник, їй потрібно залишити box-sizing,
+  який вказав розробник. Тому я використовую наслідування.
+*/
+
+* {
+  box-sizing: inherit;
+}
+
+.body {
+  margin: 0;
+}
+
+.link-reset {
+  color: inherit;
+  text-decoration: none;
+}
+
+.list-reset {
+  margin: 0;
+  padding: 0;
+
+  /*
+    If we remove the visual list markers
+    from an ordered or unordered list with list-style-type: none;,
+    we’re likely to run into some issues with VoiceOver,
+    Apple’s screen reader software, so there is invisible marker below
+  */
+
+  list-style: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'/%3E");
+  line-height: 1;
+}
+
+.container {
+  max-width: 1200px;
+
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  display: block;
+
+  height: 40px;
+  width: 40px;
+
+  object-fit: contain;
+}
+
+.nav__list {
+  display: flex;
+
+  height: 100%;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 100%;
+  padding: 24px 0;
+
+  color: #000;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-align: center;
+
+  transition: color 0.3s ease;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+
+  transition: color 0.3s ease;
```

Comment:
It's enough to just add it to class styles, it will work every time when `color` property is changed